middleware/metrics_reporter.go

```diff
@@ -18,7 +16,7 @@ var MetricTypes = struct {
 
 // MetricsReporter interface
 type MetricsReporter interface {
-	Timing(metric string, value time.Duration, tags ...string) error
+	Distribution(metric string, value float64, tags ...string) error
```

Comment:
Do we want this to be a breaking change? Could we `Distribution` as a new method?

Reply:
sure, makes sense :) done